app/routes/first-time/eligibility/claim/bank-account-details.js

```diff
@@ -1,5 +1,6 @@
-var validator = require('../../../../services/validators/payment/bank-account-details-validator')
-const bankAccountDetails = require('../../../../services/data/bank-account-details')
+const BankAccountDetails = require('../../../../services/domain/bank-account-details')
+const bankAccountDetailsData = require('../../../../services/data/bank-account-details')
```

Comment:
rename query object to represent it's operation, in this case `insert-bank-account-details-for-claim`, change to only expose that function